Base/QTCore/qSlicerCoreApplication.h

```diff
@@ -74,6 +74,7 @@ class Q_SLICER_BASE_QTCORE_EXPORT qSlicerCoreApplication : public QApplication
   Q_PROPERTY(QString slicerSharePath READ slicerSharePath CONSTANT)
   Q_PROPERTY(QString temporaryPath READ temporaryPath WRITE setTemporaryPath)
   Q_PROPERTY(QString cachePath READ cachePath WRITE setCachePath)
+  Q_PROPERTY(QString startupWorkingPath READ startupWorkingPath)
```

Comment:
```suggestion
  Q_PROPERTY(QString startupWorkingPath READ startupWorkingPath CONSTANT)
```

---- 

_The presence of the CONSTANT attribute indicates that the property value is constant. For a given object instance, the READ method of a constant property must return the same value every time it is called. This constant value may be different for different instances of the object. A constant property cannot have a WRITE method or a NOTIFY signal._

Source: https://doc.qt.io/qt-5/properties.html